tests/test-locale-assets-enqueued.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Test locale set to enqueue assets
+ *
+ * @package Caldera_Forms
+ * @since 1.8.7
+ * @license   GPL-2.0+
+ */
+class TestLocaleRenderAssets extends Caldera_Forms_Test_Case
+{
+    /**
+     * Test a locale which corresponding file exists within assets/js/i18n/
+     * 
+     * @since 1.8.7
+     *
+     * @covers  \Caldera_Forms_Render_Assets::set_locale_code($locale)
+     * @covers  \Caldera_Forms_Render_Assets::get_validator_locale_url($locale)
+     */
+    public function testKnownLocale()
+    {
+        
+        $locale = "fr";
+        $code =   Caldera_Forms_Render_Assets::set_locale_code($locale);
+        $validator_url = Caldera_Forms_Render_Assets::get_validator_locale_url($locale);
+
+        $this->assertSame( $code, "fr" );
+        $this->assertSame( $validator_url, "http://example.org/wp-content/plugins/app/assets/js/i18n/fr.js" );
```

Comment:
@New0 I would be careful about assuming site_url. Maybe better to check if the string ends with `i18n/fr.js`.